.changeset/many-baboons-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Fixed: `declaration-property-value-no-unknown` false positives via [`css-tree` 3.0.0](https://github.com/csstree/csstree/blob/v3.0.0/CHANGELOG.md#300-september-11-2024)
```

Comment:
[suggestion] Thanks for adding the changelog. I have the additional suggestions below:

- According to <https://github.com/stylelint/stylelint/pull/7993#issuecomment-2346465057>, how about including "negatives* too?
- Not a strong opinion, but how about using the npm format (`<name>@<version>`)?
- How about linking to the release page instead of the changelog heading?

```suggestion
Fixed: `declaration-property-value-no-unknown` false negatives/positives via [`css-tree@3.0.0`](https://github.com/csstree/csstree/releases/tag/v3.0.0)
```